src/components/Testimonials/index.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import Header from './../../components/Header';
+import { Container, Row, Col } from 'reactstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Testimonial from "../../components/Testimonial";
+
+export default class Testimonials extends Component {
+    render() {
+      return <Container><Col xs="auto">
+        <Header label="Create Something Amazing"></Header>
+        <div className="testimonial-box">
+          <Testimonial image="/images/testimonials/kyle.JPG" quotee="Keyle Atwood, attendee" body="It was great to see so many people passionate about designing interesting projects! Everyone was willing to help each other, including people from other teams, providing a great sense of community"></Testimonial>
```

Comment:
My name is ever-so-slightly misspelt hahaha

Reply:
Whoops, hey I'm just following the design, maybe your name needs to be changed for marketing purposes 🤣